### PR TITLE
Gevos/documentation

### PR DESCRIPTION
**Pull request by gevos**

### CODE REVIEW

File: src/README.md [hunk 1]
```diff
@@ -4,12 +4,3 @@
 | y          | y coordinate (EPSG:3857)       | y=5950684                                                    |
```
Comment (gevos) [inline src/README.md]: @mhaertwig Where should I move this documentation of URL parameters?

File: src/layers/RouteLayer/RouteLayer.js [hunk 2]
```diff
@@ -6,16 +6,16 @@ import { Style, Stroke as StrokeStyle } from 'ol/style';
...
 /**
- * Layer for visualizing fare networks.
+ * Layer for visualizing routes.
  * Extends {@link https://react-spatial.geops.de/docjs.html#vectorlayer geops-spatial/layers/VectorLayer}
  * @class RouteLayer
  * @param {Object} [options] Layer options.
- * @param {string?} [options.apiKey] Access key for geOps services.
- * @param {string} [options.name] Layer name.
- * @param {string} [options.url] Url of the geOps route backend.
+ * @param {string?} [options.apiKey] Access key for [geOps services](https://developer.geops.io/).
```
Comment (gevos) [inline src/layers/RouteLayer/RouteLayer.js]: @mhaertwig Isn't the key required?

File: src/apps/Casa/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 #
 
-This example shows a TrafimageMaps application together with layers for routing
-and fare networks.
+Casa Map is a TrafimageMaps application which is offered as a commercial map application by the SBB.
```
Comment (mhaertwig) [inline src/apps/Casa/README.md]: Casa is a sub module of `trafimage-maps` that is integrated in the SBB selling application. It is used for visualizing the planned route with different means of transportation and allows the selection of relevant fare networks.

File: src/apps/TrafimageWKP/README.md [hunk 4]
```diff
@@ -1,14 +1,8 @@
 #
 
-Configure your own map.
-
-'TrajservLayer' Layer filter options (case insensitive):
+The Trafimage Web Map Portal is used to illustrate topics relating to public transport.
```
Comment (mhaertwig) [inline src/apps/TrafimageWKP/README.md]: Trafimage maps are used to illustrate a whole range of topics relating to public transport ([more information](https://www.sbb.ch/en/bahnhof-services/bahnhoefe/karten-bahnhofplaene/trafimage-karten.html)). 
This section shows you how to configure your own map for integrating it in your existing web page.

File: src/layers/PassagierfrequenzenLayer/PassagierfrequenzenLayer.js [hunk 5]
```diff
@@ -7,6 +7,14 @@ import OLGeoJSON from 'ol/format/GeoJSON';
 import CONF from '../../config/appConfig';
 import layerHelper from '../layerHelper';
 
+/**
+ * Layer for visualizing the number of people who use the railway.
```
Comment (mhaertwig) [inline src/layers/PassagierfrequenzenLayer/PassagierfrequenzenLayer.js]: Layer for visualizing passenger frequencies of various stations.

File: src/popups/BahnhofplanPopup/README.md [hunk 6]
```diff
@@ -1,10 +1,6 @@
 #
 
-This example shows a TrafimageMaps application together with layers for routing
-and fare networks.
-
-The positioning of labels inside the fare networks layer are optimized based on
-the map extent and the used resolution.
+This example shows a popup for the two station plan layers 'Interactive Station Plan' and 'Print Products'.
```
Comment (mhaertwig) [inline src/popups/BahnhofplanPopup/README.md]: Please remove the popup-section completely.

File: src/layers/RouteLayer/RouteLayer.js (hunk 2 above)
Comment (mhaertwig) [reply src/layers/RouteLayer/RouteLayer.js]: It is

**Comment by mhaertwig** [comment]
@gevos The documentation of the TrafimageMaps parameters disappeared. I don't know when this happened but it's pretty important to have them in the documentation!

File: styleguide.config.js [hunk 7]
```diff
@@ -23,34 +23,17 @@ module.exports = {
...
       name: 'Applications',
+      content: 'src/apps/README.md',
       sections: [
         {
-          name: 'Casa',
-          content: 'src/apps/Casa/README.md',
-        },
-        {
-          name: 'Trafimage WKP',
+          name: 'Trafimage Web Map Portal',
```
Comment (mhaertwig) [inline styleguide.config.js]: Please change to "Trafimage Map Portal"

**Comment by mhaertwig** [comment]
The popups are independent from the layer documentation. Please remove them.

**Comment by gevos** [comment]
> The popups are independent from the layer documentation. Please remove them.

I have replaced the image by an image without a popup. However, to indicate the invisible layer, the image shows a pointer.

File: src/layers/RouteLayer/RouteLayer.js [hunk 8]
```diff
@@ -180,7 +180,7 @@ class RouteLayer extends VectorLayer {
   /**
```
[the remark this answers is not in the document]
Comment (oterral) [reply src/layers/RouteLayer/RouteLayer.js]: No idea

**Pull request merged**